### PR TITLE
set hostname on different OS-es

### PR DESCRIPTION
**Pull request by adevoss**
Hostname is only set on Raspberry Pi.
Maybe a cosmetic thing only.
Maybe related to #2813

### CODE REVIEW

**Comment by adevoss** [comment]
Another option is to not using `raspi-config` for setting the hostname

File: home.admin/_provision_.sh [hunk 1]
```diff
@@ -728,9 +728,16 @@ if [ ${#hostname} -gt 0 ]; then
     if [ ${#setnetworkname} -eq 0 ]; then
       setnetworkname=1
     fi
-    if [ "${setnetworkname}" = "1" ]; then
+    if [ "${setnetworkname}" == "1" ]; then
       echo "Setting new network hostname '$hostnameSanatized'" >> ${logFile}
-      sudo raspi-config nonint do_hostname ${hostnameSanatized} >> ${logFile} 2>&1
+      if [ "${baseimage}" == "raspbian" ]; then
```
Comment (rootzoll) [inline home.admin/_provision_.sh]: the standard baseimage on raspberrypi is "raspios_arm64"

**Review by rootzoll — CHANGES_REQUESTED** [review]

**Comment by adevoss** [comment]
Made the change.
Should I have approved your request? Don't see such an option in the GitHub web interface.

**Comment by adevoss** [comment]
Tested fresh install with `dev` branch.
Hostname is changed on my Debian VM.
Edit: Change was not made in (one of the) files.

**Pull request merged**